packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/react/configmid.md

```diff
@@ -1,12 +1,11 @@
-The Okta React SDK requires an instance of an `OktaAuth` object with configuration properties. Set the `clientId` and `issuer` properties with the values that you got from the CLI earlier. This can happen by directly setting the properties, with variable replacement that happens as part of the build process, or during app load time.
+The Okta React SDK requires an instance of an `OktaAuth` object with configuration properties. Set the `clientId` and `issuer` properties with the values from the CLI earlier. This can happen by directly setting the properties, with variable replacement that happens as part of the build process, or during app load time.
 
-1. Update `src/App.js` to configure Okta with the following code, replacing the placeholder values with your own values:
+1. Update `src/App.jsx` to configure Okta with the following code, replacing the `issuer` and `clientId` placeholder values with your own values (see [Finding your config values](/docs/guides/sign-into-spa-redirect/react/main/#find-your-config-values)):
```

Comment:
This example requires a Custom AS for for the section where it calls the messages API, so we'll need the instructions on setting it up somewhere